### PR TITLE
BAU: fix null place name bug

### PR DESCRIPTION
**Pull request by Joel-Pearce**
Fixes bug whereby a null place name caused an error.

- [ ] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [ ] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### CODE REVIEW

File: tests/validation_tests/test_pre_checks.py [hunk 1]
```diff
@@ -279,3 +279,28 @@ def test_full_pre_transformation_validation_pipeline_failure_place_vs_fund(valid
...
+        "",
+        "",
+        "",
+        "",
+        "",
+        "Future_High_Street_Fund",
+        "",
+        "",
+        "",
+    ]
```
Comment (cyrusdobbs) [inline tests/validation_tests/test_pre_checks.py:295]: Should we add a test for if both are null?

File: tests/validation_tests/test_pre_checks.py [hunk 2]
```diff
@@ -279,3 +279,28 @@ def test_full_pre_transformation_validation_pipeline_failure_place_vs_fund(valid
             expected_values={"TD"},
         )
     ]
+
+
+def test_full_pre_transformation_validation_pipeline_failure_place_null(valid_workbook_round_four):
+    valid_workbook_round_four["2 - Project Admin"][4] = [
```
Comment (cyrusdobbs) [inline tests/validation_tests/test_pre_checks.py:285]: These tests could also be a nice candidate for parametrisation like:

```
@pytest.fixture
def mocked_project_admin_sheet(valid_workbook_round_four, request):
    valid_workbook_round_four["2 - Project Admin"][4] = [
        "",
        "",
        "",
        "",
        "",
        request.param["fund"],
        request.param["place"],
        "",
        "",
    ]
    return valid_workbook_round_four


@pytest.mark.parametrize("mocked_project_admin_sheet, expected",
                         [({"fund": "Future_High_Street_Fund", "place": "Newark"}, [
                             WrongInputFailure(
                                 value_descriptor="Place Name vs Fund Type",
                                 entered_value="HS",
                                 expected_values={"TD"},
                             )
                         ]),
                          ({"fund": "Future_High_Street_Fund", "place": ""}, [
                              WrongInputFailure(
                                  value_descriptor="Place Name",
                                  entered_value="",
                                  expected_values=TF_PLACE_NAMES_TO_ORGANISATIONS.keys(),
                              )
                          ])], indirect=["mocked_project_admin_sheet"])
def test_full_pre_transformation_validation_pipeline_failure_place_vs_fund(mocked_project_admin_sheet, expected):
    with pytest.raises(ValidationError) as e:
        validate_before_transformation(mocked_project_admin_sheet, 4, ["Newark"])
    assert e.value.validation_failures == expected

```

**Review by cyrusdobbs — COMMENTED** [review]
Left a couple of thoughts

File: tests/validation_tests/test_pre_checks.py (hunk 1 above)
Comment (Joel-Pearce) [reply tests/validation_tests/test_pre_checks.py:295]: I don't think it makes a difference to this specific bug as if fund_type is None the failure won't arise; however, if place_name is None (or not valid), then the look-up of PLACE_NAME_TO_FUND_TYPE will result in None, and whilst ```is None in []" won't raise an error, ```is "TD" in None``` will

However, I don't think it will hurt, and good to have the coverage whilst making this change anyway. Will add

File: tests/validation_tests/test_pre_checks.py (hunk 2 above)
Comment (Joel-Pearce) [reply tests/validation_tests/test_pre_checks.py]: I've slept on pytest.parameterize, used the above and added the null Fund Type case

File: tests/validation_tests/test_pre_checks.py [hunk 3]
```diff
@@ -256,26 +256,59 @@ def test_full_pre_transformation_validation_pipeline_failure(valid_workbook):
...
+    [
+        (
+            {"fund": "Future_High_Street_Fund", "place": "Newark"},
+            [
+                WrongInputFailure(
+                    value_descriptor="Place Name vs Fund Type",
+                    entered_value="HS",
+                    expected_values={"TD"},
+                )
+            ],
```
Comment (cyrusdobbs) [inline tests/validation_tests/test_pre_checks.py:286]: I think it's worth adding a comment to explain that Newark is only valid for TD

Comment (Joel-Pearce) [reply tests/validation_tests/test_pre_checks.py:286]: Added and pushed

**Review by cyrusdobbs — APPROVED** [review]
LGTM 👍
